### PR TITLE
[CLApp] Including int for printing process

### PR DESCRIPTION
**Pull request by atfard**
In this PR, the capability of printing int-type variable is added to the print_info_in_file_process

### CODE REVIEW

**Review by loumalouomega — COMMENTED** [review]
Looks okay,@roigcarlo?

**Comment by AlejandroCornejo** [comment]
@atfard discovered that I did not expect to print int values... He need it for printing number of cycles.

File: kratos/python_scripts/print_info_in_file_process.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ def PrintOutput(self):
         else:
             for elem in self.model_part.Elements:
                 array_values = self.GetValueToPrint(elem)[self.integration_point]
-                if not isinstance(array_values, float): # Can be an scalar entity...
+                if not isinstance(array_values, float) and not isinstance(array_values, int): # Can be an scalar entity...
```
Comment (philbucher) [inline kratos/python_scripts/print_info_in_file_process.py]:
```suggestion
                if not isinstance(array_values, (float, int)): # Can be an scalar entity...
```

Comment (atfard) [reply kratos/python_scripts/print_info_in_file_process.py]: Changes applied

**Review by philbucher — APPROVED** [review]

**Pull request merged**